packages/hash/external-services/kratos/kratos.prod.yml

```diff
@@ -71,14 +68,23 @@ selfservice:
                     in: header
             - hook: session
 
-    # Email sending diabled in production for now
     verification:
       use: code
       lifespan: 48h
       # Set though SELFSERVICE_FLOWS_VERIFICATION_UI_URL
       ui_url: http://localhost:3000/verification
       enabled: true
 
+    recovery:
+      enabled: true
+      # Set through SELFSERVICE_FLOWS_RECOVERY_UI_URL
+      ui_url: http://localhost:3000/recovery
+      use: code
+
+    settings:
+      # Set through SELFSERVICE_FLOWS_SETTINGS_UI_URL
+      ui_url: http://localhost:3000/settings
```

Comment:
I think we could do this, and I think this would be a better way to go about it. Would you be okay with me doing this as a drive-by for all the URLs?

A hidden benefit of this is that we'd get errors in kratos if the URL isn't provided as an env var, so it wouldn't fallback to an invalid value

Reply:
Yes, I think this is good to do as a drive-by 👍

Reply:
Done in 6693b4b4937a84df2c7bca159c6a9d9aa72cb757 and tested locally (with a mock .env file and locally built docker container)
![image](https://user-images.githubusercontent.com/6988668/210551345-663b0ad8-25fb-497f-a1c8-dbe746dd11d8.png)

No config validation errors, just DB connection problems as I wasn't using a valid DB. All validation happens before connections start, so this should be all good.